cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -598,10 +598,9 @@ class RGate(cirq.PhasedXPowGate):
 
     def __init__(self, theta: cirq.TParamVal, phi: cirq.TParamVal) -> None:
         """Args:
```

Comment:
I know we are inconsistent with docstrings but https://google.github.io/styleguide/pyguide.html#384-classes suggests we write a one sentence summary after the `__init__`

```python
def __init__(self, likes_spam: bool = False):
        """Initializes the instance based on spam preference.

        Args:
          likes_spam: Defines if instance exhibits this preference.
        """
        self.likes_spam = likes_spam
        self.eggs = 0
```

Reply:
Yes, see https://github.com/Infleqtion/client-superstaq/pull/585